tox.ini

```diff
@@ -9,7 +9,7 @@ skipsdist = True
 
 [testenv]
 install_command =  pip install {opts} {packages}
-commands = py.test --cov=matrix {posargs} -v --ignore=tests/functional
+commands = py.test --cov=matrix {posargs} -v --ignore=tests/functional --ignore=lib --ignore=bin --ignore=local
```

Comment:
Tests run perfectly with this. Yay.

Please add the same invocation to the "functional" env as well.